### PR TITLE
Fix bad shim namespaces

### PR DESCRIPTION
**Pull request by blairconrad**
From https://github.com/FakeItEasy/FakeItEasy/pull/620#issuecomment-190708031.

### CODE REVIEW

File: tests/FakeItEasy.Net35.Tests/ConditionalWeakTableTests.cs [hunk 1]
```diff
@@ -21,19 +26,38 @@ public void Should_release_value_when_there_are_no_more_references()
...
             GC.Collect();
 
+            // Assert
             TypeWithWeakReference result = null;
-            Assert.That(table.TryGetValue(strong, out result), Is.True);
-            Assert.That(result, Is.SameAs(weak));
+            table.TryGetValue(strong, out result).Should().BeTrue();
+            result.Should().BeSameAs(weak);
+        }
+
```
Comment (blairconrad) [inline tests/FakeItEasy.Net35.Tests/ConditionalWeakTableTests.cs]: Seemed like the original test was 2 tests, so I split while I was here.

File: tests/FakeItEasy.Net35.Tests/ZipTests.cs [hunk 2]
```diff
@@ -15,12 +19,10 @@ public void Should_zip_correctly()
...
             // Act
-            var zipped = letters.Zip(numbers, (f, s) => new { Letter = f, Number = s }).ToList();
+            var zipped = letters.Zip(numbers, (l, n) => l + n);
 
             // Assert
-            Assert.That(zipped, Has.Count.EqualTo(2));
-            Assert.That(zipped[0], Has.Property("Letter").EqualTo("a").And.Property("Number").EqualTo(1));
-            Assert.That(zipped[1], Has.Property("Letter").EqualTo("b").And.Property("Number").EqualTo(2));
+            zipped.ShouldBeEquivalentTo(new[] { "a1", "b2" });
         }
```
Comment (blairconrad) [inline tests/FakeItEasy.Net35.Tests/ZipTests.cs]: The reference redirect thingy made `.ToList()` inaccessible. Or hard to access. When I eyeballed it, I thought, "that's more complicated than it maybe needs to be". So…

File: tests/FakeItEasy.Net35.Tests/ConditionalWeakTableTests.cs (hunk 1 above)
Comment (adamralph) [inline tests/FakeItEasy.Net35.Tests/ConditionalWeakTableTests.cs]: :+1: agreed. Great stuff

File: tests/FakeItEasy.Net35.Tests/ZipTests.cs (hunk 2 above)
Comment (adamralph) [inline tests/FakeItEasy.Net35.Tests/ZipTests.cs]: Nice simplification!

**Comment by blairconrad** [comment]
Thanks. And for label.

**Pull request merged**